frontend/src/app/game/simulation-control/simulation-control.component.html

```diff
@@ -9,7 +9,7 @@
     <cvd-row wrap>
       <cvd-button (click)="gameService.togglePause()" [color]="speed === 'pause' ? 'red' : 'blue'">{{speed === 'pause' ?
         'Resume' : 'Pause'}}</cvd-button>
-      <cvd-button (click)="gameService.restartSimulation('max')">Compute</cvd-button>
+      <cvd-button (click)="gameService.setSpeed('max')">Compute</cvd-button>
```

Comment:
Do you get the charts updated? I think you should `this.updateChart('all')` in setSpeed/max branch.